### ISSUE TITLE
Error on creating new Multi app

### ISSUE THREAD

**Issue by chrismessina** [kofigumbs/multi#51]
Here are the settings I used to generate a new app:

```
{
  "tabs": [
    {
      "title": "Substack Reader",
      "url": "https://reader.substack.com"
    }
  ]
}
```

When Terminal opened up, there was an error on creation, although the app was created successfully:

![CleanShot 2020-12-16 at 11 19 36@2x](https://user-images.githubusercontent.com/22610/102396377-3c55dd00-3f91-11eb-8987-9d4d103dd5ac.png)

**Comment by kofigumbs** [comment]
It looks like this might be a Big Sur issue. It should only occur when you create a new Multi app, and it doesn't affect the integrity of the newly created app. Looking into workarounds.

Note to self: https://developer.apple.com/forums/thread/666939

**Comment by chrismessina** [comment]
As I mentioned, this might be a sandboxing issue: 
https://stackoverflow.com/questions/9989561/os-x-sandboxing-lsopenfromurlspec-returned-10827

The error `NSOSStatusErrorDomain Code=-10827` shows up around sandboxing.

**Comment by kofigumbs** [comment]
The fix for this is released in [2.1.0](https://github.com/kofigumbs/multi/releases/tag/v2.1.0).
